src/utils.py

```diff
@@ -38,6 +38,7 @@
 DEFAULT_PUBSUB_TO_IOT_CORE = 'False'
 
 GG_CORE_NAME = os.environ.get("AWS_IOT_THING_NAME")
+GG_ROOT_CA_PATH = os.environ.get("GG_ROOT_CA_PATH")

```

Comment:
how do we handle this if the environment variable isn't set, as it won't be on older nucleus versions.

Reply:
In that case, it would just pass None : 
1. if https_proxy was configured: component would fail
2. if http_proxy or no proxy was configured, it should be ok
Either we log that, so customer can update nucleus maybe if they are using proxy?

Reply:
What about when there's no proxy involved?

does setting the proxy_ca_bundle have an adverse affect?

Reply:
No, I tried the case when there was no proxy configured and we pass in proxy config, which succeeds without error.
Also, I looked at the code, if `proxy_url` is None (as in this case, where we wont have any values for http_proxy and https_proxy). It takes the non proxy path.